Carmu/Sources/Models/DayOfWeek.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  DayOfWeek.swift
+//  Carmu
+//
+//  Created by 김동현 on 11/7/23.
+//
+
+import Foundation
+
+enum DayOfWeek: Int {
+    case mon = 0
+    case tue = 1
+    case wed = 2
+    case thu = 3
+    case fri = 4
+    case sat = 5
+    case sun = 6
+
+    static let weekday: Set<DayOfWeek> = [.mon, .tue, .wed, .thu, .fri]
+    static let weekend: Set<DayOfWeek> = [.sat, .sun]
+    static let everyday: Set<DayOfWeek> = [.mon, .tue, .wed, .thu, .fri, .sat, .sun]
+
```

Comment:
```suggestion
enum DayOfWeek: Int, CaseIterable {
    case mon = 0
    case tue = 1
    case wed = 2
    case thu = 3
    case fri = 4
    case sat = 5
    case sun = 6

    static let weekday: Set<DayOfWeek> = [.mon, .tue, .wed, .thu, .fri]
    static let weekend: Set<DayOfWeek> = [.sat, .sun]
}
```
CaseIterable 프로토콜을 채택하면 열거형의 모든 값의 배열을 allCases 속성으로 받아올 수 있습니다!
everyday는 따로 안만드셔도 될 것 같아요.

사용은 이런 식으로.. 👉 `DayOfWeek.allCases`